src/app/static-pages/team-page/team-member/team-member.component.scss

```diff
@@ -0,0 +1,78 @@
+@use '../../../../assets/scss/fonts' as fonts;
+
+.element-titel {
+  color: #160637;
+  padding: 0 0 10px 0;
+}
+.row-member{
+  display: inline-block;
+  background: #F8F8F8;
+  padding: 20px 10px;
+  box-sizing: border-box;
+  border-radius: 30px;
+  height: auto;
+  margin: 0 0 25px;
+  p{
+    font-size: 16px;
+  }
+  h3 {
+    @include fonts.font__pt-mono--bold;
+  }
+  @media screen and (max-width: 479px) {
+    width: 100%;
+  }
+  @media screen and (min-width: 480px) and (max-width: 783px) {
+    width: 260px;
+  }
+  @media screen and (min-width: 784px) and (max-width: 1023px) {
+    width: 310px;
+  }
```

Comment:
Die erste Media Query sollte nicht notwendig sein.
Und bei den folgenden zwei Queries kannst du das `and (max-width: ...)` entfernen, weil ja die folgende Media Query schon den nächsten Breakpoint definiert.

Reply:
Die erste Media Query habe ich drin gelassen, da entsprechend die Breite der Kacheln angepasst wird.

Reply:
Das könntest du in dem Fall auch einfach ohne Media Query als Standardwert vorgeben, aber passt auch so 👍